assets/scss/global/_settings.scss

```diff
@@ -35,10 +35,11 @@
 //  30. Table
 //  31. Tabs
 //  32. Thumbnail
-//  33. Tooltip
-//  34. Top Bar
+//  33. Title Bar
+//  34. Tooltip
+//  35. Top Bar
 
-@import '../../components/foundation-sites/scss/util/util';
+@import 'util/util';
```

Comment:
@olefredrik you can import through just `util/util` because `assets/components/foundation-sites/scss` is in the Sass import paths in `gulpfile.js`. When I refactored `foundation.scss` the other day I cleaned up a bunch of the paths because the whole path wasn't necessary.